apps/endatix-hub/app/(public)/slack/page.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
```

Comment:
This can be executed at the server. Added this task to track it - [Slack Integration: Add Settings Persistence](https://github.com/endatix/endatix/issues/319#top)
#319